### ISSUE TITLE
Last merge didn't get uploaded

### ISSUE THREAD

**Issue by dpryan79** [conda-forge/root5-feedstock#26]
@xhochy @conda-forge/root5 The last PR merge didn't upload anything (or at least nothing appears in the conda-forge channel)

**Comment by xhochy** [comment]
Empty commit on master should fix things. Seems like the commit was not pushed from Github to the CI providers.

**Comment by dpryan79** [comment]
Yup, I'd do it myself if I had commit rights (not that I really need to be the maintainer of yet another feedstock).

**Comment by mbargull** [comment]
If gh-24 builds, let's merge that one.
